Annotation/declarative_golden_set.json

```diff
@@ -249,7 +273,7 @@
                     }

```

Comment:
I can't tell if "when I say xx" should be considered an EC?

Reply:
I think it's a temporal relative clause - what's being trusted is "me" and the trusting is happening at some time